external-builds/pytorch/pytorch_audio_repo.py

```diff
@@ -68,10 +58,16 @@ def add_common(command_parser: argparse.ArgumentParser):
             default=THIS_DIR / THIS_MAIN_REPO_NAME,
             help="Git repository path",
         )
```

Comment:
```suggestion
command_parser.add_argument(
"--checkout-dir",
type=Path,
default=THIS_DIR / THIS_MAIN_REPO_NAME,
help=f"""Directory path where the git repo is cloned into. Default is {THIS_DIR / THIS_MAIN_REPO_NAME}""",
)
```

can we please rename `--repo` to `--checkout-dir` and improve the description? it is absolutely not clear to me just from the description that this is the target location when the git checkout will go to
(git clone also calls it `<directory>`)